test/blocks/notification/notification.test.js

```diff
@@ -0,0 +1,72 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { setConfig } from '../../../libs/utils/utils.js';
+
+const locales = { '': { ietf: 'en-US', tk: 'hah7vzn.css' } };
+const conf = { locales };
+setConfig(conf);
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/notification/notification.js');
+
+describe('notification', () => {
+  const notifs = document.querySelectorAll('.notification');
+  notifs.forEach((notif) => init(notif));
```

Comment:
This looks prone to building up test state. It would be better to adding the notifications to the body in a `beforeEach` block and clearing the DOM In a `afterEach` block.

Tests should be isolated, debugging and changing tests that already depend on shared test state is very tedious